nginx/nginx.conf

```diff
@@ -89,8 +89,12 @@ http {
 			proxy_pass http://relay;
 		}
 		location ^~ /js-sdk/ {
-			autoindex on;
-			root  /var/www/js-sdk;
+			include mime.types;
+			autoindex off;
+			etag on;
```

Comment:
Nginx enables `etag` by default for static contents and 99.99% it keeps on doing this.